src/lib/config.js

```diff
@@ -31,12 +31,30 @@
 
 const RC = require('parse-strings-in-object')(require('rc')('ES', require('../../config/default.json')))
 
+const DEFAULT_PROTOCOL_VERSION = {
+  CONTENT: '1.1',
+  ACCEPT: {
+    DEFAULT: '1', // This is not currently used for unit test purposes, and it is here for consistency between services.
+    VALIDATELIST: [
+      '1',
+      '1.1'
+    ]
+  }
+}
+
+const T_PROTOCOL_VERSION = { ...DEFAULT_PROTOCOL_VERSION, ...RC.PROTOCOL_VERSIONS }
+if (RC.PROTOCOL_VERSIONS) T_PROTOCOL_VERSION.ACCEPT = { ...DEFAULT_PROTOCOL_VERSION.ACCEPT, ...RC.PROTOCOL_VERSIONS.ACCEPT }
+if (T_PROTOCOL_VERSION.ACCEPT && T_PROTOCOL_VERSION.ACCEPT.VALIDATELIST && (typeof T_PROTOCOL_VERSION.ACCEPT.VALIDATELIST === 'string' || T_PROTOCOL_VERSION.ACCEPT.VALIDATELIST instanceof String)) {
+  T_PROTOCOL_VERSION.ACCEPT.VALIDATELIST = JSON.parse(T_PROTOCOL_VERSION.ACCEPT.VALIDATELIST)
+}
+
```

Comment:
There is a lot of hard-to-test business logic in these lines.

Can you please make this a function such as `getProtocolVersionOrDefault()` that accepts 2 arguments: `RC.PROTOCOL_VERSIONS` and the `DEFAULT_PROTOCOL_VERSION` and returns a resolved object?

Reply:
Updated PR. Please review.